src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -564,10 +568,14 @@
 type: command
 short-summary: Update the properties of a Vault.
 examples:
-  - name: Update the properties of a Vault. (autogenerated)
+  - name: Update the properties of a key vault. (autogenerated)
```

Comment:
this isn't consistent with existing example, and one below

Reply:
@yungezz The example you mention contains a deprecated argument I remove it as you request. 

And now, I saw the nearly nowhere contains description like `a Vault` but `a key vault` appears more often. I think we no need to update this line?
Or shall we update the `a Vault` short-summary to make them consistent?

Reply:
@bim-msft , with MHSM, what's more appropriate name for a vault? Key vault or vault?

Reply:
@yungezz `Vault` is a more appropriate name.

Reply:
reverted